src/common/util/compute_rtl.ts

```diff
@@ -0,0 +1,9 @@
+import { HomeAssistant } from "../../types";
+
+export default function computeRTL(hass: HomeAssistant) {
```

Comment:
Please don't use `export default`

```suggestion
export computeRTL = (hass: HomeAssistant) => {
```